forge/routes/api/team.js

```diff
@@ -205,6 +220,9 @@ module.exports = async function (app) {
 
     /**
      * @deprecated Use /:teamId/applications, or /:applicationId/instances
+     * For now, the project link nodes depend on this endpoint. Once the
+     * project link nodes are updated to be application aware, this endpoint
+     * may be removed (assuming no other clients are using it)
```

Comment:
```suggestion
     * This end-point is still used by the project nodes and nr-tools plugin.
```
This comment is not correct. Updating the project nodes isn't sufficient - there will be a long-tail of existing installs that don't update their stacks so have older versions of the project nodes. There are also other consumers of this api - such as nr-tools-plugin to consider.

It is sufficient just to note who still consumes this deprecated endpoint. Any commentary on if/when it can be removed doesn't belong here.